src/sonic-frr/Makefile

```diff
@@ -4,20 +4,18 @@ SHELL = /bin/bash
 
 MAIN_TARGET = $(FRR)
 DERIVED_TARGET = $(FRR_PYTHONTOOLS) $(FRR_DBG) $(FRR_SNMP) $(FRR_SNMP_DBG)
-BRANCH = $(shell date +%Y%m%d\.%H%M%S)
 
 $(addprefix $(DEST)/, $(MAIN_TARGET)): $(DEST)/% :
 	# Build the package
 	pushd ./frr	
-	git checkout -b $(BRANCH) origin/frr/7.1
-	stg init
+	stg branch --create temp origin/frr/7.1
```

Comment:
Can FRR_VERSION be used instead of hardcoding the version to 7.1?

Reply:
Thanks for your comment. I will change it.